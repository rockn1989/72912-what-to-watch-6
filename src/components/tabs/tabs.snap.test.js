import React from 'react';
import {render} from '@testing-library/react';
import Tabs from './tabs';

it(`Should Review render correctly`, () => {
  const tabs = [`one`, `two`, `three`];
  const active = 0;
  const func = () => {};
  const child = [1, 2, 3];

  const {container} = render(
      <Tabs tabsTitle={tabs} onChangeActiveItem={func} activeTab={active}>
        {child}
      </Tabs>
  );
  expect(container).toMatchSnapshot();
});
