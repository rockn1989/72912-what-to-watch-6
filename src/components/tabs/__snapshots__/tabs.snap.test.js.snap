// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Review render correctly 1`] = `
<div>
  <nav
    class="movie-nav movie-card__nav"
  >
    <ul
      class="movie-nav__list"
    >
      <li
        class="movie-nav__item movie-nav__item--active"
      >
        <a
          class="movie-nav__link"
          href="#"
        >
          one
        </a>
      </li>
      <li
        class="movie-nav__item "
      >
        <a
          class="movie-nav__link"
          href="#"
        >
          two
        </a>
      </li>
      <li
        class="movie-nav__item "
      >
        <a
          class="movie-nav__link"
          href="#"
        >
          three
        </a>
      </li>
    </ul>
  </nav>
  1
</div>
`;
